src/Pages/SignIn/SignIn.js

```diff
@@ -0,0 +1,226 @@
+import React from "react";
+import { withRouter, Link } from "react-router-dom";
+import SideNav from "../../Components/Nav/SideNav/SideNav.js";
+import Footer from "../../Components/Footer/Footer.js";
+import API_URL from "../../Config.js";
+import "./SignIn.scss";
+
+class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      isLoginBtnHovered: false,
+      isSignUpBtnHovered: false,
+      userId: "",
+      userPw: "",
+      isSubmit: false,
+      isValidate: false,
+    };
+  }
+
+  handleUserInfo = (e) => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { userId, userPw } = this.state;
+    fetch(`${API_URL}/signin`, {
+      method: "POST",
+      body: JSON.stringify({
+        userid: userId,
+        password: userPw,
+      }),

```

Comment:
```suggestion
      body: JSON.stringify({ userid, password }),
```
state의 key값을 body에 담길 key값과 일치시켜준다면(백엔드에 요청을 할 수도 있겠습니다만) 객체 단축 속성명을 활용할 수 있습니다. 꼭 맞출 필요까지는 없을거 같습니다만 일치시킨다면 혹시 모를 오타 등의 문제를 피할 수 있을 것 같긴 하네요.

Reply:
상의해보겠습니다